test/common/buffer/buffer_fuzz.cc

```diff
@@ -114,6 +114,10 @@ class StringBuffer : public Buffer::Instance {
     ::memcpy(data, this->start() + start, size);
   }
 
+  uint64_t copyOutToSlices(uint64_t, Buffer::RawSlice*, uint64_t) const override {
+    PANIC("not implemented");
```

Comment:
shouldn't this be fuzzed? cc @adisuissa

Reply:
Yes, it would be great. I think CopyOut can be used as a basis for fuzzing this method: 
https://github.com/envoyproxy/envoy/blob/e845ee8faf95f21a0c4aad6af78a679ab5f79ff8/test/common/buffer/buffer_fuzz.cc#L309-L320

Reply:
I will take look at into fuzz test